Assets/_BuildingBlocks/OGCWebServices/Shared/BaseRequest.cs

```diff
@@ -105,6 +105,18 @@ protected string GetInnerTextForNode(XmlNode layerNode, string nodeName, bool se
             }
         }
 
+        //protected XmlNodeList GetNodesByName(XmlNode layerNode, string nodeName)
+        //{
+        //    var queryForNode = $"*[local-name()='{nodeName}']";
+        //    return layerNode.SelectNodes($".//{queryForNode}", namespaceManager);
+        //}
+
+        protected XmlNodeList GetNodesByName(XmlNode layerNode, string nodeName)
+        {
+            var queryForNode = $"//*[local-name()='{nodeName}' or @*='{nodeName}']";
```

Comment:
Dit gaat mogelijk onverwachte effecten geven omdat ineens een attribuut met dezelfde naam matcht; maar zoals besproken: we kunnen veel beter overgaan op serialisatie want we missen nu wel meer delen uit de specificatie dus dit gaan we nog een keer herzien